### ISSUE TITLE
Platform iOS

### ISSUE THREAD

**Issue by jaio771** [mauron85/react-native-background-geolocation#114]
Receive old locations. 

## Your Environment

* Plugin version:
* Platform: iOS 
* OS version:macOS
* Device manufacturer and model:
* React Native version:
* Plugin configuration options: 
* Link to your project:

## Context
 I need to choose how often to have background location updates (min 2 seconds, max 60 seconds). With the library itself I found that the minimum time I can have is 30seconds. I hack it and now I can get locations more frequent, but the problem is that I found that the location I receive in on ('location') is not the last one. I assume that I am doing something wrong, but I can't figured out what.
Is there any way to reduce that frequency and have always the last location?

Thanks in advance!

## Steps to Reproduce

1.
2.
3.
4.

**Comment by jaio771** [comment]
I need  to choose how often to  have background location updates (min 2 seconds, max 60 seconds). With the library itself I found that the minimum time I can have is 30seconds. I hack it and now I can get locations more frequent, but the problem is that I found that the location I receive in on ('location') is not the last one. I assume that I am doing something wrong, but I can't figured out what. 
Is there any way to reduce that frequency and have always the last location?

Thanks in advance!
